api/HTMLSourceElement.json

```diff
@@ -120,10 +120,10 @@
               "version_added": "12"
             },
             "firefox": {
-              "version_added": "3.5"
+              "version_added": "15"
```

Comment:
Hmm, what's going on here? This isn't when `<picture>` support was shipped, but it does match the media attribute data in source.json.

Since the `<source media>` attribute has been used for both `<video>` and `<picture>` and the use in the former context was removed from the spec, it would be good to have some notes around this. Not necessary for this PR though, I expect this is making the data more correct.